packages/expressive-code/README.md

```diff
@@ -50,13 +54,38 @@ const { renderedGroupAst, styles } = await ec.render({
 let htmlContent = toHtml(renderedGroupAst)
 
 // Collect styles and add them before the HTML content
-const stylesToPrepend: string[] = []
+const stylesToPrepend = []
 stylesToPrepend.push(baseStyles)
-stylesToPrepend.push(...styles)
-if (stylesToPrepend.length) {
-  htmlContent = `<style>${[...stylesToPrepend].join('')}</style>${htmlContent}`
-}
-
+stylesToPrepend.push(themeStyles)
+stylesToPrepend.push(...blockStyles)
+
+const styleContent = `<style> ${[...stylesToPrepend].join('')} </style>`
+const jsContent = `<script type="module"> ${[...jsModules].join('')} </script>`
+
+const htmlDocument = `
+<!doctype html>
+<html lang="en" data-theme="light">
```

Comment:
```suggestion
<html lang="en">
```

In the default configuration, the `data-theme` attribute takes a theme name as its value, not the type - so the allowed values in your example would be `github-dark` or `github-light`.

That being said, I'd rather try to keep this example minimal in terms of HTML and therefore suggest dropping the the `data-theme` attribute altogether. This would have the added benefit of demonstrating the default dark/light theme switching support depending on the user agent / OS preferences through EC's default media queries. Viewing a page without this attribute will make the code block automatically appear in the theme matching the user's color preferences.